### ISSUE TITLE
Allow hiding fields by configuration

### ISSUE THREAD

**Issue by amenk** [sbodak/magento2-checkout-custom-form#18]
Would you accept a pull request for configuring the fields shown on checkout via store config or do you strongly recommend to fork and customize for the use case instead?

**Comment by amenk** [comment]
CC @kolaente

**Comment by sbodak** [comment]
Hi, create PR, I will check how it looks :)

**Comment by kolaente** [comment]
PR is up: #19

**Comment by sbodak** [comment]
@kolaente I was thinking a little bit about this PR and I decided to not merge it.

The reason is that this module is still rather an example of how to create custom checkout form using `UI component` and how to change it or extend, instead of using plugin on `LayoutProcessor`.

I have a plan to build a different module which allow you to customize checkout.
https://github.com/sbodak/magento2-checkout-builder

Currently, I've released a 1.2 tag which has a few fixes for displaying form in expecting place, after shipping address and it's tested in Magento 2.3.

thanks for your work
